### PR TITLE
doc: add minutes for meeting 10 April 2024 (#1526)

### PR DESCRIPTION
**Pull request by ShogunPanda**

### CODE REVIEW

**Review by MoLow — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by ShogunPanda** [comment]
What is the policy for PRs here? When do I merge?

File: meetings/2024-04-10.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* deps,lib,src: add experimental web storage [#52435](https://github.com/nodejs/node/pull/52435)
+  * We discussed and came to conclusion to proceed with the pull-request. We have no objections for it to land, other than existing collaborator comments.
+* doc: add policy for “placeholder” executables [#52107](https://github.com/nodejs/node/pull/52107)
+  * A vote has been started - https://github.com/nodejs/TSC/pull/1527
```
Comment (styfle) [inline meetings/2024-04-10.md:45]: Is this link correct? It says "Open Corepack Vote", not "placholder vote"

Comment (mhdawson) [reply meetings/2024-04-10.md:45]: I believe that is the right link

**Comment by mhdawson** [comment]
@ShogunPanda there are a bunch of linter issues that need to be resolved.

**Comment by ShogunPanda** [comment]
@mhdawson Fixed them. Ready to go?

**Comment by mhdawson** [comment]
@ShogunPanda many thank!

**Pull request merged**